gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -257,6 +274,16 @@ typedef struct
   uint32_t dimension[NNS_TENSOR_META_RANK_LIMIT];
   uint32_t format;
   uint32_t media_type;
+
+  /**
+   * @brief Union of the required information for processing each tensor "format".
+   * @todo Implement "tensor-query"
+   */
+  union {
+    GstSparseTensorInfo sparse_info;
+    GstTensorQueryInfo query_info;
```

Comment:
Query and Sparse-Tensor are orthogonal. Why they are in a single union?

Reply:
`sparse_info` contains single integer value. Query related functions may be responsible for preserving the information.
Here Query is introduced for showing concept of format-internal data. Later we can decide whether to keep single union or to separate `query_info`.

Reply:
Because someone may "query" with "sparse tensors", they should be separated from the beginning.
You need to union mutually exclusive items of the same category; e.g., "tensor format" (default, flex, sparse).
Otherwise, they should exist independently.

Reply:
@myungjoo Right, I excluded query related code in this commit. Those will be properly added by other PRs.